client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.module.css

```diff
@@ -57,6 +57,14 @@
     /* … */
 }
 
+.results {
+  padding-left: 5px;
```

Comment:
Maybe a bit more padding on the side?
```suggestion
  padding-left: 10px;
```

Reply:
Design doc shows 20px. Let's do that, unless y'all notice an issue with it.

![PR #403 Padding](https://user-images.githubusercontent.com/62723358/119391368-e26de780-bc93-11eb-8d45-e9b50a6edc3c.png)